api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1234,6 +1236,22 @@ public static String getOpenmrsProperty(String property) {
 	public static final String LOG_LEVEL_ERROR = "error";
 	
 	public static final String LOG_LEVEL_FATAL = "fatal";
+
+	/**
+	 * Global property value which defines the default authentication
```

Comment:
GP_LOGIN_URL is well described.
But the description for LOGIN_URL is still not matching what GP_LOGIN_URL talks about.
Can you take a look an existing properties descriptions for an example?
Feel free to ask your team mates for help.

Reply:
resolved?